Gemfile.lock

```diff
@@ -63,4 +63,4 @@ DEPENDENCIES
   yard (~> 0.9)
 
 BUNDLED WITH
-   2.2.16
+   2.2.21
```

Comment:
Is this change expected?

Reply:
I assume it happened because my bundler version is newer. Would you prefer to roll this back? IMO, I'd just keep it

Reply:
Let's remove it 🔥